packages/devtools_app/lib/src/framework/release_notes/README.md

```diff
@@ -1,9 +1,9 @@
-## Writing DevTools release notes
+## Generating Release notes
 - Release notes for DevTools are hosted on the flutter website (see [archive](https://docs.flutter.dev/development/tools/devtools/release-notes)).
-- To add release notes for the latest release, create a PR with the appropriate changes for your release
-    - The [release notes template](release-notes-template.md) can be used as a starting point
-    - see example [PR](https://github.com/flutter/website/pull/6791).
+- To add release notes for the latest release, create a PR with the appropriate changes for your release.
+    - To generate the release notes run:
 
+        `dart tool/release_note_helper.dart markdown -f tool/release_notes/release_notes.json`
 - Test these changes locally before creating the PR.
```

Comment:
the rest of these instructions are obsolete now that we will no longer be pushing the code to the flutter website ourselves

Reply:
Talked with Kenzie:
change the instructions to :
- create an issue of flutter/website
- add the markdown to the issu